smurfs/src/components/SmurfList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Smurf from './Smurf';
+
+const SmurfList = props => {
```

Comment:
It may make a little more sense to use connect in this component rather than App